expression/aggregation.go

```diff
@@ -108,7 +110,14 @@ func NewAggFunction(funcType string, funcArgs []Expression, distinct bool) Aggre
 	case ast.AggFuncAvg:
 		return &avgFunction{aggFunction: newAggFunc(tp, funcArgs, distinct)}
 	case ast.AggFuncGroupConcat:
-		return &concatFunction{aggFunction: newAggFunc(tp, funcArgs, distinct)}
+		var sep string
+		for _, opt := range concatOpt {
```

Comment:
Is it better to use  the first element of concatOpt as sep and the second element as orderby?  @coocood What's your opinion?